### PR TITLE
OData Release Note and Documentation

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/partners/sap/sap-destination-service.md [hunk 1]
```diff
@@ -40,6 +40,17 @@ The **Get Destination** action properties look like this:
 
 **Variable** is the name of the object of type *SAPODataConnector.Destination* where the details of the destination will be stored.
 
+### 4.1 Overriding XSUAA Destination
```
Comment (gauravkrshah) [inline content/partners/sap/sap-destination-service.md:43]: This change is related to accessing the SAP Destination service. Let's change the title to "Accessing SAP Destination Service"

File: content/partners/sap/sap-destination-service.md [hunk 2]
```diff
@@ -40,6 +40,17 @@ The **Get Destination** action properties look like this:
...
+
+By default, your destination will be found and authenticated using XSUAA. This is controlled using a constant `XSUAAEnabled` which defaults to `true`.
```
Comment (gauravkrshah) [inline content/partners/sap/sap-destination-service.md:45]: This section requires changes:
In any case whether XSUAAEnabled set as true or false. XSUAA service is used to get authentication token/access token to call/access destination service.
Difference in both the cases are:
1. Flag set to true (Default):
Application is using XSUAA service for user authentication, which generates an access token using user authentication credentials. This access token will then be used to authenticate with destination service.
2. Flag set to false:
a. Application not using XSUAA for user authentication
b. Application do not need/want to user access token for destination service authentication. 

Both a and b only applicable when SAP Destination is configured as "No Authentication" or "Basic Authentication"

File: content/releasenotes/developer-portal/sap-cloud-platform.md [hunk 3]
```diff
@@ -14,6 +14,13 @@ For information on the current status of deployment to SAP BTP and any planned r
 
 ### July 26th, 2021
 
+#### Improvements – OData Connector for SAP Solutions
+
+* We improved the error handling for expired access tokens when using [Get Destination](/partners/sap/sap-destination-service). Users will now be signed out so that they can retrieve a new access token.
+* We added a setting to override XSUAA authentication and allow you to use basic or no authentication for your destination instead.
```
Comment (gauravkrshah) [inline content/releasenotes/developer-portal/sap-cloud-platform.md:20]: Added ability to fetch SAP Destination even if the application is not XSUAA enabled or not using XSUAA

**Review by gauravkrshah — CHANGES_REQUESTED** [review]

File: content/partners/sap/sap-destination-service.md (hunk 1 above)
Comment (MarkvanMents) [reply content/partners/sap/sap-destination-service.md:43]: How about "Authenticating SAP Destination Service"?

Comment (MarkvanMents) [reply content/partners/sap/sap-destination-service.md]: Changed in this commit: https://github.com/mendix/docs/pull/3600/commits/992577fdb6605d35cdacccd3322c46d20bd41a9c

File: content/partners/sap/sap-destination-service.md (hunk 2 above)
Comment (MarkvanMents) [reply content/partners/sap/sap-destination-service.md:45]: Changed in this commit: https://github.com/mendix/docs/pull/3600/commits/992577fdb6605d35cdacccd3322c46d20bd41a9c

File: content/releasenotes/developer-portal/sap-cloud-platform.md (hunk 3 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/sap-cloud-platform.md]: Changed in this commit: https://github.com/mendix/docs/pull/3600/commits/992577fdb6605d35cdacccd3322c46d20bd41a9c

**Comment by MarkvanMents** [comment]
@gauravkrshah 
Hi Gaurav,
I've updated the PR based on your comments. Can you let me know if you are happy with the changes or if there are any other changes which would improve the documentation.
Mark

**Review by gauravkrshah — APPROVED** [review]
Looks Good.

**Comment by MarkvanMents** [comment]
Thanks - let me know when this can be published.

**Pull request merged**